slices.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'redcarpet'         , '~> 3.0.0'
   s.add_dependency 'RedCloth'          , '~> 4.2.9'
   s.add_dependency 'stringex'          , '~> 1.4.0'
-  s.add_dependency 'will_paginate'     , '3.0.pre4'
+  s.add_dependency 'will_paginate_mongoid', '~> 2.0.1'
```

Comment:
That's what we've been doing with the other gems? but I'll make the change and thanks to #118 will find out if it breaks